.github/ISSUE_TEMPLATE/page-report-zh-cn.yml

```diff
@@ -5,58 +5,58 @@ body:
   - type: markdown
     attributes:
       value: |
-        ### Before you start
+        ### 在提交议题前
 
-        **Want to change this page yourself?** This content is open source!
-        ↩ Go back and use the _Edit on GitHub_ link on the page.
+        **想要自行修改页面？** 文档内容是开源的！
+        ↩ 返回，并点击页面中的 _Edit the page on Github_ 链接。
 
-        **Is your issue about the browser compatibility table?**
-        ↩ Go back and use the _Report problems with this compatibility data on GitHub_ link on the page.
+        **你的议题有关于浏览器兼容性表格？**
+        ↩ 返回，并点击页面中的 _Report problems with this compatibility data on GitHub_ 链接。
 
-        **Need help with a browser?**
-        🙋 To get help with [Firefox](https://support.mozilla.org/en-US/kb/file-bug-report-or-feature-request-mozilla), [Chrome](https://support.google.com/chrome/answer/95315?hl=en-GB&ref_topic=7439544), [Safari](https://www.apple.com/feedback/safari.html), or another browser, check the browser's support site.
+        **需要与浏览器相关的帮助？**
+        🙋 请访问浏览器支持站点，以获取有关 [Firefox](https://support.mozilla.org/kb/file-bug-report-or-feature-request-mozilla)、[Chrome](https://support.google.com/chrome/answer/95315?ref_topic=7439544)、[Safari](https://www.apple.com/feedback/safari.html) 或其它浏览器的帮助。
```

Comment:
```suggestion
        🙋 请访问浏览器支持站点，以获取有关 [Firefox](https://support.mozilla.org/kb/file-bug-report-or-feature-request-mozilla)、[Chrome](https://support.google.com/chrome/answer/95315?ref_topic=7439544)、[Safari](https://www.apple.com/feedback/safari.html) 或其它浏览器的帮助信息。
```

Reply:
The support site of Apple Safari is https://support.apple.com/safari.
For SUMO, may we link to zh-CN(https://support.mozilla.org/zh-CN/kb/file-bug-report-or-feature-request-mozilla)?

Reply:
Yes, I just followed https://github.com/mdn/translated-content/pull/8538#issuecomment-1260856502 before.

Reply:
> The support site of Apple Safari is https://support.apple.com/safari.

I prefer this one. Could you open a PR for (at least) [this file](https://github.com/mdn/translated-content/blob/main/.github/ISSUE_TEMPLATE/page-report.yml)

Reply:
> > The support site of Apple Safari is https://support.apple.com/safari.
> 
> ~I prefer this one. Could you open a PR for (at least) [this file](https://github.com/mdn/translated-content/blob/main/.github/ISSUE_TEMPLATE/page-report.yml)~
> 
> Edit: it seems that we are all locating to feedback pages

Oh, I see.